recipes/newrelic/apm/dotNet/linux-systemd.yml

```diff
@@ -18,6 +18,26 @@ keywords:
 
 processMatch: []
 
+preInstall:
+  requireAtDiscovery:
+    - |
+      rm -rf /tmp/dotnet-introspector >/dev/null
```

Comment:
One this we have been trying to get better at is store files in non deterministic location.  See the java recipe for an example:
https://github.com/newrelic/open-install-library/blob/main/recipes/newrelic/apm/java/linux.yml#L33